### PR TITLE
added another feature; closes #3

### LINKED ISSUE #3: Another feature proposal
Added a description

### PR DESCRIPTION
**Pull request by jpreitsma**

### CODE REVIEW

File: the-other-feature-description.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Beef soft tacos
+Ingredients
+seasoned beef
+Soft corn tortillas
+salsa
+guacamole
+Shredded Mexican cheese
+Shredded lettuce
+Sour cream
+Method
```
Comment (juholehtonen) [inline the-other-feature-description.md:10]: There could be empty line before 'Method'

File: the-other-feature-description.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Prepare seasoned beef using recipe at seasoned beef, or just reheat remaining beef from previous meal.
+
+Heating the tortillas: For melted cheese, spread tortillas in single layer and sprinkle cheese on each tortilla. Heat tortillas for 25 sec in microwave, then cover with foil.
+
+Assemble tacos using all ingredients.
```
Comment (juholehtonen) [inline the-other-feature-description.md:15]: This is good.

File: the-other-feature-description.md (hunk 1 above)
Comment (jpreitsma) [reply the-other-feature-description.md:10]: True!

File: the-other-feature-description.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Serve with beans and/or rice on side.
```
Comment (juholehtonen) [inline the-other-feature-description.md:17]: Looking good. I also experiment with different commenting/reviewing buttons in this PR view

**Review by juholehtonen — APPROVED** [review]
Very nice!

**Pull request merged**